### PR TITLE
Add option for job list cutoff

### PR DESCRIPTION
**Pull request by ingkebil**
So we can see more than 10 jobs.

### CODE REVIEW

File: moneypenny/flow/cli.py [hunk 1]
```diff
@@ -17,15 +17,21 @@ def flow(context):
...
+@click.option('--limit', default=10,
+              help='only show the top "limit" jobs')
+def ls(context, all, limit):
     """List all the jobs."""
     jobs = context.obj['db'].queued()
     count = jobs.count()
     if count == 0:
         click.echo("no jobs queued")
+    elif all:
+        limit = -1
```
Comment (robinandeer) [inline moneypenny/flow/cli.py]: what does this actually mean?

**Comment by robinandeer** [comment]
Very nice improvement!

File: moneypenny/flow/cli.py (hunk 1 above)
Comment (ingkebil) [inline moneypenny/flow/cli.py]: No limit. The SQLAlchemy docs didn't have any documentation on this.

But SO did: http://stackoverflow.com/questions/12836194/how-do-i-remove-a-limit-that-is-already-applied-to-a-query-object

Maybe I should set it to `None` instead? Makes it more readable.

**Comment by ingkebil** [comment]
Ready for merge?

**Comment by robinandeer** [comment]
yes go ahead!

**Pull request merged**